### PR TITLE
Website loading speed improvements.

### PR DESCRIPTION
**Pull request by ajayyy**
- Made the website not wait for the image to start the animations (The image was 5 mb).
- Slightly lowered size of image (now only 4k). A 5 megabyte 6k image is a little overkill.

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -25,10 +25,8 @@
 		});
 
 	// Play initial animations on page load.
-		$window.on('load', function() {
-			window.setTimeout(function() {
-				$body.removeClass('is-preload');
-			}, 500);
+		$(document).ready(function() {
```
Comment (NikhilPeri) [inline assets/js/main.js:28]: This is simple and soo much better !!

**Review by NikhilPeri — COMMENTED** [review]
This is great! Thank you for fixing it :) 

I wasnt able to verify that the fix worked ... I am requesting the page locally through `python -m SimplteHTTPServer` and have chrome dev tools set to a custom profile with `100000 kb/s` down 

It seems like we still wait for `IMG_0513` to load before playing the animation ... I mean this is better than waiting for all to load, but I was wondering if you knew why or if this is just something wrong with my setup

**Comment by ajayyy** [comment]
From my testing, the animation started while the background was black. I had to lower the resolution since, otherwise, the animation would stutter when the image finally loaded. Maybe it would be a good idea to lower the resolution a little more (it is still very large), but there was no stuttering for me at this size. I haven't tested if it stutters on a phone.

**Comment by ajayyy** [comment]
Okay, it looks like the animation still waits for the image, but it at least places the shape down before the image appears.

**Comment by ajayyy** [comment]
I'm not sure of a good way of fixing that issue other than to switch out how animations work. Right now, they are transitions. The body starts with a class called `is-preload`, which has certain values set. Then, the class is removed in JS to trigger the transition. Instead, a CSS animation with keyframes can be used, which I think might not have these issues. Then, using an `ease-out` animation curve instead of an `ease-in` animation curve could help make it feel less delayed as well.

Though, that isn't completely necessary to do, as I feel this fixes it to be usable, which is all that is really needed.

Since the scripts are below the body, they are loaded after it. Because of this, the images are loaded first. It has to be after the body loads to change the body class and keep the original transition state.

Because of this, there are two options that can fix this.

- Use keyframe animations
- "Lazy load" images by having all their sources set in JS

Both of these solutions seem unnecessary for the little extra functionality it adds.

**Review by buzuloiu — COMMENTED** [review]
This is cool! How were you testing the loading times? I mean intuitively it makes sense and looks good to me, I'm just curious as to how you did it. Did you host it somewhere?

**Comment by ajayyy** [comment]
I was just loading it from my local disk, no http server even. The load times were still noticeable. Chrome allows you to limit your network speed now in the inspect element options, which can help.

image

**Review by buzuloiu — APPROVED** [review]

**Pull request merged**